iis-common/src/test/java/eu/dnetlib/iis/common/spark/TestWithSharedSparkSession.java

```diff
@@ -26,6 +28,7 @@ public void beforeEach() {
             SparkConf conf = new SparkConf();
             conf.setMaster("local");
             conf.set("spark.driver.host", "localhost");
+            conf.set("spark.serializer", "org.apache.spark.serializer.KryoSerializer");
```

Comment:
Not sure this is crucial but after we refactor the tests we will be missing the following line:

`conf.setAppName(${someTestClassName}.class.getSimpleName());`

Reply:
Forgot about that, so I'm adding this conf parameter. Funny thing, but when creating the same functionality for `SparkContext` in similar class an error was thrown because application name was missing.